src/lib/header-bidding/prebid-types.ts

```diff
@@ -125,7 +129,8 @@ export type BidderCode =
 	| 'sonobi'
 	| 'triplelift'
 	| 'trustx'
-	| 'xhb';
+	| 'xhb'
+	| 'kargo';
```

Comment:
Super nitpicky this one, but could we keep these codes in alphabetical order? Same with the prebid params below as well.

Reply:
how do we feel about enabling various alphabetical eslint rules?
https://typescript-eslint.io/rules/sort-type-constituents/

For another PR I think though!

Reply:
> how do we feel about enabling various alphabetical eslint rules?

I'd defer to the rest of the team on this one. I tend to prefer alphabetical ordering in these situation when there's no obvious precedence / order of importance, however when there is a more obvious ordering then I prefer keeping it explicit.

Reply:
I agree with Chris. I like alphabetical ordering _except_ when another ordering makes more sense. I wouldn't want to have to disable the eslint rule when another order makes more sense.